app/assets/stylesheets/common.scss

```diff
@@ -753,6 +753,14 @@ header .search_forms,
   display: none;
 }
 
+[dir=ltr] input#query {
```

Comment:
As I thought `r2` will take of the RTL stuff so there's no need to do it yourself.

Also please move this inside the `.search_form` block below - as it is it will match any input element with that name anywhere on the site.

Reply:
Thank you, I just fixed it. 👍